[2023-07-28T14:46:27.760+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcp_extract_and_load_incremental.gcp_load_incremental manual__2023-07-28T14:40:33.006878+00:00 [queued]>
[2023-07-28T14:46:27.781+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcp_extract_and_load_incremental.gcp_load_incremental manual__2023-07-28T14:40:33.006878+00:00 [queued]>
[2023-07-28T14:46:27.782+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-07-28T14:46:27.852+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): gcp_load_incremental> on 2023-07-28 14:40:33.006878+00:00
[2023-07-28T14:46:27.863+0000] {standard_task_runner.py:57} INFO - Started process 9335 to run task
[2023-07-28T14:46:27.875+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'gcp_extract_and_load_incremental', 'gcp_load_incremental', 'manual__2023-07-28T14:40:33.006878+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/incremental_copy.py', '--cfg-path', '/tmp/tmpmbwsca4f']
[2023-07-28T14:46:27.883+0000] {standard_task_runner.py:85} INFO - Job 239: Subtask gcp_load_incremental
[2023-07-28T14:46:28.116+0000] {task_command.py:410} INFO - Running <TaskInstance: gcp_extract_and_load_incremental.gcp_load_incremental manual__2023-07-28T14:40:33.006878+00:00 [running]> on host a7b633085ee1
[2023-07-28T14:46:28.396+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcp_extract_and_load_incremental' AIRFLOW_CTX_TASK_ID='gcp_load_incremental' AIRFLOW_CTX_EXECUTION_DATE='2023-07-28T14:40:33.006878+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-28T14:40:33.006878+00:00'
[2023-07-28T14:46:28.740+0000] {logging_mixin.py:150} INFO - Importing 4 rows to marine-fusion-394105.employees_db.employees...
[2023-07-28T14:46:28.741+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/incremental_copy.py", line 139, in gcp_load_incremental
    job = client.load_table_from_json(data, table_id, job_config=job_config)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 2845, in load_table_from_json
    data_str = "\n".join(json.dumps(item, ensure_ascii=False) for item in json_rows)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 2845, in <genexpr>
    data_str = "\n".join(json.dumps(item, ensure_ascii=False) for item in json_rows)
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type date is not JSON serializable
[2023-07-28T14:46:28.768+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=gcp_extract_and_load_incremental, task_id=gcp_load_incremental, execution_date=20230728T144033, start_date=20230728T144627, end_date=20230728T144628
[2023-07-28T14:46:28.829+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 239 for task gcp_load_incremental (Object of type date is not JSON serializable; 9335)
[2023-07-28T14:46:28.857+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-28T14:46:28.971+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
